index.js

```diff
@@ -172,8 +172,8 @@ function log(instanceLog) {
     instanceLog = initPlugins()(instanceLog);
   }
 
-  return function (level, msg, data) {
-    data = data || {};
+  return function (level, msg, data = {}) {
+    data = Object.assign({}, data);
```

Comment:
This will only work for properties whose values are primitives. As painful as it is, we may want to use: 
`data = JSON.parse(JSON.stringify(data));` to clone without preserving any object references.